test/e2e/apigatewayv2/tests/helper.py

```diff
@@ -0,0 +1,209 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Helper functions for apigatewayv2 tests
+"""
+from apigatewayv2 import CRD_GROUP, CRD_VERSION, SERVICE_NAME
+from common import k8s
+from common.resources import load_resource_file
+
+API_RESOURCE_PLURAL = 'apis'
+INTEGRATION_RESOURCE_PLURAL = 'integrations'
+AUTHORIZER_RESOURCE_PLURAL = 'authorizers'
+ROUTE_RESOURCE_PLURAL = 'routes'
+STAGE_RESOURCE_PLURAL = 'stages'
+
+
+def api_ref_and_data(api_resource_name: str, replacement_values: dict):
```

Comment:
For most of the resource loading methods in this directory, if they're being used for fixtures, you can probably re-use the same abstraction we created for the SageMaker resources.
We created a generic "load resource" method - https://github.com/aws-controllers-k8s/community/blob/main/test/e2e/common/k8s.py#L111
We then override it with the SageMaker defaults - https://github.com/aws-controllers-k8s/community/blob/main/test/e2e/sagemaker/__init__.py#L30

This should save time dealing with silly repetitive code.

Reply:
I will need to pull latest changes from main branch for this. I will try it tomorrow.

Reply:
@RedbackThomson I think @vijtrip2 can do that in a followup PR. I'd like to merge this PR soon and Vijay can improve it in later ones.